sentences/fr/homeassistant_HassTimerStatus.yaml

```diff
@@ -6,56 +6,60 @@ intents:
           # Il reste combien de temps sur le minuteur de la cuisine
-          - "Il reste combien [de temps] (sur|dans) [(le|mon)] <minuteur> [<de>] [<le>]{area}"
+          - "[Il reste] [encore] combien [de temps] (sur|dans) [(le|mon)] <minuteur> [<de>] [<le>]{area}"
         expansion_rules:
           reste_t_il: "(reste-t-il)|(reste t il)"
 
       # duration
       - sentences:
           # Combien de temps reste-t-il au minuteur de 2 minutes
-          - "Combien de temps <reste_t_il> au <minuteur> [de] <timer_start>"
+          - "[Encore] Combien [de temps] <reste_t_il> au <minuteur> [de] <timer_start>"
           # Combien de temps reste-t-il sur le minuteur de 2 minutes
-          - "Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur> [de] <timer_start>"
+          - "[Encore] Combien [de temps] <reste_t_il> (sur|dans) [(le|mon)] <minuteur> [de] <timer_start>"
           # Il reste combien de temps au minuteur de 2 minutes
-          - "Il reste combien [de temps] au <minuteur> [de] <timer_start>"
+          - "[Il reste] [encore] combien [de temps] au <minuteur> [de] <timer_start>"
           # Il reste combien de temps sur le minuteur de 2 minutes
-          - "Il reste combien [de temps] (sur|dans) [(le|mon)] <minuteur> [de] <timer_start>"
+          - "[Il reste] [encore] combien [de temps] (sur|dans) [(le|mon)] <minuteur> [de] <timer_start>"
         expansion_rules:
           reste_t_il: "(reste-t-il)|(reste t il)"
 
       # name
       - sentences:
           # Combien de temps reste-t-il au minuteur appelé Pizza
-          - "Combien de temps <reste_t_il> au <minuteur> [<appele>] {timer_name:name}"
+          - "[Encore] Combien [de temps] <reste_t_il> au <minuteur> [<appele>] [<le>] {timer_name:name}"
           # Combien de temps reste-t-il sur le minuteur appelé Pizza
-          - "Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur> [<appele>] {timer_name:name}"
+          - "[Encore] Combien [de temps] <reste_t_il> (sur|dans) [(le|mon)] <minuteur> [<appele>] [<le>] {timer_name:name}"
+          # Combien de temps reste-t-il pour la pizza
+          - "[Encore] Combien [de temps] <reste_t_il> [<appele>] [<le>] {timer_name:name}"
           # Il reste combien de temps au minuteur appelé Pizza
-          - "Il reste combien [de temps] au <minuteur> [<appele>] {timer_name:name}"
+          - "[Il reste] [encore] combien [de temps] au <minuteur> [<appele>] [<le>] {timer_name:name}"
           # Il reste combien de temps sur le minuteur appelé Pizza
-          - "Il reste combien [de temps] (sur|dans) [(le|mon)] <minuteur> [<appele>] {timer_name:name}"
+          - "[Il reste] [encore] combien [de temps] (sur|dans) [(le|mon)] <minuteur> [<appele>] [<le>] {timer_name:name}"
+          # Il reste combien de temps pour la pizza / Encore combien pour la pizza
+          - "[Il reste] [encore] combien [de temps] [<appele>] [<le>] {timer_name:name}"
```

Comment:
For all these sentences, split them so that each of them have at least
- "Il reste" (or `<reste_t_il>` it's the same thing)
or
- "de temps"

To avoid having too short sentences such as "Combien"

So, for exmaple, this sentence:

```yaml
- "[Il reste] [encore] combien [de temps]"
```

Could become
```yaml
- "Il reste [encore] combien [de temps]"
- "[Il reste] [encore] combien de temps"
```